CHANGELOG.md

```diff
@@ -21,6 +21,9 @@
 
 ## Bug Fixes
 
+- Disabled IREE on MacOS due to compatibility issues and added the CasADI
+  path to the environment to resolve issues on MacOS and Linux. Windows
+  users may still experience issues with interpolation. ([#4528](https://github.com/pybamm-team/PyBaMM/pull/4528))
```

Comment:
It's not a big deal, but removing `IREE` should be under breaking changes, right?

Reply:
I put it there because it was probably not used. It was only enabled for a single python version, so unlikely that anyone will be using it. I will put the full removal of IREE under breaking changes when I do that work.